src/ert/storage/local_ensemble.py

```diff
@@ -552,6 +552,9 @@ def save_parameters(
                 "Dataset uses 'float64' for fields/surfaces. Use 'float32' to save memory."
             )
 
+        if True in xr.DataArray.isnull(dataset["values"]):
```

Comment:
This will happen: https://github.com/equinor/ert/blob/9ad807253b3183cda424aac6e22f9ec1b5ad4579/src/ert/analysis/_es_update.py#L721, so think it makes sense to do the check there (or perhaps both places), then we can also recommend an alternative (using subspace instead of excact inversion)

Reply:
Or maybe catch the error there, and reraise with more context?

Reply:
Checking for NANs when saving in _es_update will cause fields to fail because we store inactive cells as nans.